File: src/app/model/act.ts
import {ActType} from './actType';
import {Phase} from './phase';

export class Act {
  roundId: number;
  userId: string;
  playerId: number;
  roomId: number;
  type: ActType;
  phase: Phase;
  bet: number;
  nextUserId: string;
}

File: app/src/main/java/be/kdg/mobile_client/model/Act.java
package be.kdg.mobile_client.model;

import lombok.Data;

@Data
public class Act {
    private int roundId;
    private String userId;
    private int playerId;
    private int roomId;
    private ActType type;
    private Phase phase;
    private int bet;
    private int totalBet;
    private String nextUserId;
}
